src/main/java/org/opentripplanner/graph_builder/module/PruneIslands.java

```diff
@@ -62,7 +65,17 @@ public class PruneNoThruIslands implements GraphBuilderModule {
    */
   private int pruningThresholdIslandWithStops;
 
-  public PruneNoThruIslands(
+  /**
+   * search radius as meters when looking for island neighbours
+   */
+  private static double adaptivePruningDistance;
+
+  /**
+   * Coefficient how much larger islands (compared to max sizes defined above) get pruned if they are close enough
+   */
+  private static double adaptivePruningFactor;
```

Comment:
These two variables must not be static.

In fact we are moving away from setters at all and pass all of the parameters in the constructor.

Reply:
Fixed. Use of static methods and attributes was some sort of historical remainder.